### PR TITLE
docs(guide): Deprecated react-instansearch

### PR DESCRIPTION
**Pull request by JureSotosek**
In the [Guides Migration Guide - Package import](https://community.algolia.com/react-instantsearch/guide/Migration_guide_package_import.html) it is stated that "The package react-instantsearch is deprecated in favor of react-instantsearch-dom and react-instantsearch-native." while the [guide](https://community.algolia.com/react-instantsearch/Getting_started.html#install-react-instantsearch) still instructs you to install the "react-instantsearch" package and not the "-dom" one.

I also changed the "Note:" part for it to be consistent with the [React-Native](https://community.algolia.com/react-instantsearch/Getting_started_React_native.html) guide.

### CODE REVIEW

File: docgen/src/Getting_started.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ It contains sample data for an e-commerce website.
...
-React InstantSearch is available in the [npm](https://www.npmjs.com) registry. Install it:
+React InstantSearch is available in the [npm](https://www.npmjs.com) registry. To install it:
 
 ```shell
-yarn add react-instantsearch
+yarn add react-instantsearch-dom
 ```
 
-Note: we use `yarn add` to install dependencies but React InstantSearch is also installable via `npm install`.
+Note: we use `yarn add` to install dependencies but React InstantSearch is also installable via `npm install --save react-instantsearch-dom`.
```
Comment (francoischalifour) [inline docgen/src/Getting_started.md]: The flag `--save` is implicit with npm now, you can remove it.

Comment (JureSotosek) [reply docgen/src/Getting_started.md]: Do i remove only the `--save` tag or also the `react-instantsearch-dom` from the change?

Comment (francoischalifour) [reply docgen/src/Getting_started.md]: Only the `--save` tag. That'd be awesome!

**Comment by JureSotosek** [comment]
I guess i will also remove the flag in the react-native guide since it is present there also.

**Comment by francoischalifour** [comment]
@JureSotosek It's not a big deal but feel free to do so!

**Pull request merged**